### ISSUE TITLE
BatchNorm 

### ISSUE THREAD

**Issue by liu666666** [BVLC/caffe#5705]
@pluskid  @sergeyk @Franck-Dernoncourt @gut 
Thanks for your nice work!
I have two questions:
1.The value of batch size in train.prototxt is 24, I use 4 gpus with nccl, and the value of iter_size in solver.prototxt is 2. So my real batch_size is 2442=192.Right?
2.The function of BatchNorm layer is to normalise the input to have 0-mean and/or unit (1) variance across the batch. what is the batch in your code If I have set the parameters as mentioned in (1)? 24 or 96 or 192?

**Comment by gut** [comment]
I guess GitHub's issue tracker is not the place for that. Please ask on, e.g, StackOverflow.
Also avoid pinging devs directly (why me?)

**Comment by shaibagon** [comment]
Please use the [caffe-users list][1] for usage, installation, or modeling questions, or other requests for help.
You may also post questions on [stackoverflow][3], make sure you tag them with `caffe` tag.  
There is also [caffe.help][4] documenting the different layers of caffe.
Do not post such requests to Issues. Doing so interferes with the development of Caffe.

Please read the [guidelines for contributing][2] before submitting this issue.

 [1]: https://groups.google.com/forum/#!forum/caffe-users
 [2]: https://github.com/BVLC/caffe/blob/master/CONTRIBUTING.md
 [3]: http://stackoverflow.com/questions/tagged/caffe
 [4]: http://caffe.help/manual/layers.html
